Makefile

```diff
@@ -13,7 +14,7 @@ build:
 
 push:
 	docker buildx build \
-          --platform linux/amd64,linux/arm64 \
+          --platform $(PLATFORMS) \
```

Comment:
I don't think we should add it here as a configurable variable. You could accidentally push it without all the platforms.

Reply:
Spot on, thank you for noticing this!